src/components/10-atoms/toggle-switch/react/ControlledWithDummyOnChange.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AXAToggleSwitchReact from './AXAToggleSwitchReact';
+
+export default class example extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      active: true,
+      receivedEvent: '-',
+    };
+  }
+
+  change = ev => {
+    this.setState({ active: ev.target.checked });
+    this.setState({ receivedEvent: String(ev.target.checked) });
+  };
+
+  render() {
+    return (
+      <div>
+        <AXAToggleSwitchReact
+          active={this.state.active}
```

Comment:
No need to put this into state, wouldn't a constant be enough, like in the reproduction sandbox code? This here makes it more tricky to reason about things.

Reply:
Using state for this rerenders the component differently, if `active` would not stay at the same value. Having that in `states` ensures that the `onChange` handler does not misbehave.